amgut/db/patches/0041.sql

```diff
@@ -208,60 +262,231 @@ CREATE INDEX idx_dna_plate_extraction_robot_id ON pm.dna_plate ( extraction_robo
+    plate_type_id               bigint  NOT NULL,
+    volume                      real  NOT NULL,
+    dna_quantification_date     timestamp  ,
+    dna_quantification_email    varchar  ,
+    dna_quantification_volume   real  ,
+    plate_reader_id             bigint  ,
+    CONSTRAINT pk_whole_genome_plate PRIMARY KEY ( whole_genome_plate_id ),
+    CONSTRAINT idx_whole_genome_plate UNIQUE ( name ),
+    CONSTRAINT fk_whole_genome_plate FOREIGN KEY ( email ) REFERENCES ag.labadmin_users( email ),
+    CONSTRAINT fk_whole_genome_plate_prc_robot FOREIGN KEY ( processing_robot_id ) REFERENCES pm.processing_robot( processing_robot_id ),
+    CONSTRAINT fk_whole_genome_plate_type FOREIGN KEY ( plate_type_id ) REFERENCES pm.plate_type( plate_type_id ),
+    CONSTRAINT fk_whole_genome_plate_email FOREIGN KEY ( dna_quantification_email ) REFERENCES ag.labadmin_users( email ),
+    CONSTRAINT fk_whole_genome_plate_reader FOREIGN KEY ( plate_reader_id ) REFERENCES pm.plate_reader( plate_reader_id )
+ ) ;
+CREATE INDEX idx_whole_genome_plate_0 ON pm.whole_genome_plate ( email ) ;
+CREATE INDEX idx_whole_genome_plate_1 ON pm.whole_genome_plate ( processing_robot_id ) ;
+CREATE INDEX idx_whole_genome_plate_2 ON pm.whole_genome_plate ( plate_type_id ) ;
+CREATE INDEX idx_whole_genome_plate_3 ON pm.whole_genome_plate ( dna_quantification_email ) ;
+CREATE INDEX idx_whole_genome_plate_4 ON pm.whole_genome_plate ( plate_reader_id ) ;
+
+CREATE TABLE pm.condensed_plates (
+    whole_genome_plate_id  bigint  NOT NULL,
+    dna_plate_id           bigint  NOT NULL,
+    position               integer  NOT NULL,
+    CONSTRAINT idx_condensed_plates PRIMARY KEY ( whole_genome_plate_id, dna_plate_id, position ),
+    CONSTRAINT fk_condensed_plates FOREIGN KEY ( whole_genome_plate_id ) REFERENCES pm.whole_genome_plate( whole_genome_plate_id ),
+    CONSTRAINT fk_condensed_plates_dna_plate FOREIGN KEY ( dna_plate_id ) REFERENCES pm.dna_plate( dna_plate_id )
+ ) ;
+CREATE INDEX idx_condensed_plates_0 ON pm.condensed_plates ( whole_genome_plate_id ) ;
+CREATE INDEX idx_condensed_plates_1 ON pm.condensed_plates ( dna_plate_id ) ;
+
+CREATE TABLE pm.whole_genome_plate_layout (
+    whole_genome_plate_id bigint  NOT NULL,
+    sample_id             varchar  ,
+    row                   integer  NOT NULL,
+    col                   integer  NOT NULL,
+    name                  varchar  ,
+    notes                 varchar  ,
+    dna_concentration     real,
+    CONSTRAINT fk_whole_genome_plate_layout FOREIGN KEY ( whole_genome_plate_id ) REFERENCES pm.whole_genome_plate( whole_genome_plate_id ),
+    CONSTRAINT fk_whole_genome_plate_layout_sample FOREIGN KEY ( sample_id ) REFERENCES pm.sample( sample_id )
+ ) ;
+CREATE INDEX idx_whole_genome_plate_layout ON pm.whole_genome_plate_layout ( whole_genome_plate_id ) ;
+CREATE INDEX idx_whole_genome_plate_layout_0 ON pm.whole_genome_plate_layout ( sample_id ) ;
+
+CREATE TABLE pm.wgs_library_prep_kit (
```

Comment:
I think that they may be better called "shotgun", in comparison to "target". Because other than whole-genome, people can do whole-transcriptome, whole-exsome, etc using the same sequencing protocol (but only the sample processing protocol gets changed). "shotgun" and "target" (or "targetted") best describe the nature of the protocols (random or specific primers).